CRM/Contribute/WorkflowMessage/ContributionTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/**
+ * @method array getContribution()
+ * @method array getContact()
+ * @method $this setContact(array $contact)
```

Comment:
Agree this PR looks safe / consistent / starting-place for other revisions. :+1:

I was about to hit "Merge" but then wondered about this fragment - `contact` is a property of `GenericWorkflowMessage` not `ContributionTrait`. 

There is another property in this trait, `contributionId` -- should it reference that instead?

```
 * @method ?int getContributionId()
 * @method $this setContributionId(?int $contributionId)
```